cabal.project

```diff
@@ -9,7 +9,7 @@ packages:
          ./hls-plugin-api
          ./hls-test-utils
          ./plugins/hls-cabal-fmt-plugin
-         ./plugins/hls-tactics-plugin
+         ./plugins/hls-refine-destruct-plugin
```

Comment:
I suggest we don't do this - we do publish these to hackage, so renaming them is just a bit unnecessarily annoying.